Source/WebCore/css/CSSBasicShapes.cpp

```diff
@@ -300,6 +311,72 @@ String CSSXywhValue::customCSSText() const
         bottomRightRadiusWidth, bottomRightRadiusHeight, bottomLeftRadiusWidth, bottomLeftRadiusHeight);
 }
 
+CSSRectShapeValue::CSSRectShapeValue(Ref<CSSValue> top, Ref<CSSValue> right, Ref<CSSValue> bottom, Ref<CSSValue> left, RefPtr<CSSValue> topLeftRadius, RefPtr<CSSValue> topRightRadius, RefPtr<CSSValue> bottomRightRadius, RefPtr<CSSValue> bottomLeftRadius)
+    : CSSValue(RectShapeClass)
+    , m_top(WTFMove(top))
```

Comment:
If you WTFMove these arguments shouldn't they be passed as r-value references?

Reply:
Fixed